src/Shell/Listener/TinkerEventEmitter.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Laravel\Tinker\Shell\Listener;
+
+use Psy\ExecutionLoop\AbstractListener;
+use Psy\Shell;
+
+class TinkerEventEmitter extends AbstractListener
+{
+    public function afterLoop(Shell $shell)
+    {
+        event('tinker.after-loop');
+    }
+
+    public static function isSupported()
+    {
+        return true;
```

Comment:
?

Reply:
This is used to tell PsyShell that we want to use this Listener. See https://github.com/bobthecow/psysh/blob/master/src/ExecutionLoop/Listener.php#L22 for more details.

Should I add a comment here to clarify this? As we do not want to conditionally deactivate this listener I just return `true` here.